files/helpers/const.py

```diff
@@ -606,6 +606,6 @@
 
 procoins_li = (0,2500,5000,10000,25000,50000,125000,250000)
 
-linefeeds_regex = re.compile("([^\n])\n([^\n])", flags=re.A)
+linefeeds_regex = re.compile("([^\n0-9])\n([^\n0-9])", flags=re.A)
```

Comment:
Alright, question!

The original regex matched solo newlines, i.e. a newline bordered by things that aren't newlines. The new regex is a similar match but adding numbers to the mix. However, this doesn't mean it's "between lines that begin with a number", it means "between lines, as long as the first character of the second line isn't a number, *or* the *last* character of the first line isn't a number".

I feel like this should have some kind of bug I should be able to trigger by ending lines with numbers but I haven't been able to do so; however I think you understand this code better than I do.

Maybe it's OK to just use `([^\n])\n([^\n0-9])`?

Or maybe I'm just wrong on this, let me know :V

Reply:
After thinking about this one a bit, I think I'm gonna agree with Zorba that skipping the newline doubling when the previous line ends with a number is probably not what we want to do here.

Reply:
Good points, thanks for the feedback. 
I'm back to work so I'll need a few days to address this, but I won't forget this.

Reply:
No worries, thanks for working on it :)

Reply:
@zorbathut I updated the PR to go with the solution proposed by TLSM in https://github.com/themotte/rDrama/issues/143#issuecomment-1200451493 since the forum went live and since I don't have a better idea how to change this without upsetting other formatting code.

Thanks for your patience.

Reply:
Forgot to attach screenshot of results of changes:

![themotte_unbusted_lists](https://user-images.githubusercontent.com/109989267/190034344-46c53a71-d541-473b-bd6f-1b8f9c00e649.png)